airflow/upgrade/rules/logging_configuration.py

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import absolute_import
+
+from airflow.configuration import conf
+from airflow.upgrade.rules.base_rule import BaseRule
+
+
+class LoggingConfigurationRule(BaseRule):
+    title = "Logging configuration has been moved to new section"
+
+    description = "The logging configurations have been moved from [core] to the new [logging] section."
+
+    def check(self):
+        if not conf.has_option("logging", "logging_level"):
+            return (
+                "The following configurations have been to moved from [core] to the new [logging] section. \n"
+                "- base_log_folder \n"
+                "- remote_logging \n"
+                "- remote_log_conn_id \n"
+                "- remote_base_log_folder \n"
+                "- encrypt_s3_logs \n"
+                "- logging_level \n"
+                "- fab_logging_level \n"
+                "- logging_config_class \n"
+                "- colored_console_log \n"
+                "- colored_log_format \n"
+                "- colored_formatter_class \n"
+                "- log_format \n"
+                "- simple_log_format \n"
+                "- task_log_prefix_template \n"
+                "- log_filename_template \n"
+                "- log_processor_filename_template \n"
+                "- dag_processor_manager_log_location \n"
+                "- task_log_reader \n"
```

Comment:
I would suggest checking if any of this key is used by users (not empty or different than default - this can be tricky) and then print a message for each incompatibility. See #11056 for reference. In this way, the message will be more precise. What do you think?

Reply:
Ah I was not thinking of this cause this change is backwards compatible and not something that can break if not moved but 
you are right, showing only the incompatibility really helps while taking actions, I can do a comparison and return only the actively used keys but what of the default/ empty keys is it right that they still remain in the old section ?

Reply:
> what of the default/ empty keys is it right that they still remain in the old section ?

I think it will be ok to hardcode them in rule using default values from `airflow.cfg` of 1.10.x. WDYT @kaxil ?

Reply:
Yea default values can be hardcoded.

And also agree that showing a precise message for the used config will be better